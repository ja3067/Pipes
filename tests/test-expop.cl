x = 3 ** 4
print(x)

print(4.0 *  .5)

x += 5
print(x)
