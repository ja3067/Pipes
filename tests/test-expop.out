81
2
86